### PR TITLE
Working with environment section

### PR DESCRIPTION
**Pull request by SafeEHA**
Added Working with environment section to the Tryit.md page on the user-guide 
This is the final part of the [#337](https://github.com/metal3-io/metal3-io.github.io/issues/337) issue

### CODE REVIEW

File: docs/user-guide/src/developer_environment/tryit.md [hunk 1]
```diff
@@ -103,3 +103,501 @@ To set environment variables persistently, export them from the configuration fi
  cp config_example.sh config_$(whoami).sh
  vim config_$(whoami).sh
 ```
+
+## 2. Working with Environment
```
Comment (elfosardo) [inline docs/user-guide/src/developer_environment/tryit.md:107]: let's change this to "Working with the Development Environment"

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: docs/user-guide/src/developer_environment/tryit.md [hunk 2]
```diff
@@ -103,3 +103,501 @@ To set environment variables persistently, export them from the configuration fi
...
+ Id   Name       State
+--------------------------
+ 1    minikube   running
+ 2    node_0     running
+ 3    node_1     running
+```
+
+Each of the VMs (aside from the `minikube` management cluster VM) is
+represented by `BareMetalHost` objects in our management cluster. The yaml
+definition file used to create these host objects is in `bmhosts_crs.yaml`.
```
Comment (hroyrh) [inline docs/user-guide/src/developer_environment/tryit.md:154]: I cannot find the file `bmhosts_crs.yaml`, can you please check if it exists or the name is different ?

Comment (SafeEHA) [reply docs/user-guide/src/developer_environment/tryit.md:154]: I can't find it too, i've searched for it on the user-guide but so far the only place it has appeared on is 03_launch_mgmt_cluster.sh script

File: docs/user-guide/src/developer_environment/tryit.md [hunk 3]
```diff
@@ -103,3 +103,501 @@ To set environment variables persistently, export them from the configuration fi
...
+      name: node-0-bmc-secret
+      namespace: metal3
+    credentialsVersion: "1789"
+```
+
+### 2.2. Provision Cluster and Machines
+
+This section describes how to trigger the provisioning of a cluster and hosts via
+`Machine` objects as part of the Cluster API integration. This uses Cluster API
+[v1beta1](https://github.com/kubernetes-sigs/cluster-api/tree/v1.0.2) and
```
[the remark this answers is not in the document]
Comment (SafeEHA) [reply docs/user-guide/src/developer_environment/tryit.md:284]: So, i should replace that link with this?

File: docs/user-guide/src/developer_environment/tryit.md [hunk 4]
```diff
@@ -103,3 +103,502 @@ To set environment variables persistently, export them from the configuration fi
...
+assumes that metal3-dev-env is deployed with the environment variable
+**CAPM3_VERSION** set to **v1beta1**. This is the default behaviour. The v1beta1 deployment can be done with
+Ubuntu 20.04 or Centos 8 Stream target host images. Please make sure to meet
```
Comment (hroyrh) [inline docs/user-guide/src/developer_environment/tryit.md:286]: Let's use the latest supported versions for Ubuntu and Centos here.

File: docs/user-guide/src/developer_environment/tryit.md [hunk 5]
```diff
@@ -103,3 +103,502 @@ To set environment variables persistently, export them from the configuration fi
...
+ Id   Name       State
+--------------------------
+ 1    minikube   running
+ 2    node_0     running
+ 3    node_1     running
+```
+
+Each of the VMs (aside from the `minikube` management cluster VM) is
+represented by `BareMetalHost` objects in our management cluster. The yaml
+
```
Comment (hroyrh) [inline docs/user-guide/src/developer_environment/tryit.md:154]: A part of the line seems to be missing here.

File: .vscode/settings.json [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (elfosardo) [inline .vscode/settings.json:2]: this is vscode config that needs to be removed from the PR

**Review by elfosardo — CHANGES_REQUESTED** [review]

**Pull request merged**